src/loaders.ts

```diff
@@ -33,6 +55,27 @@ export class EtherscanABILoader implements ABILoader {
     this.baseURL = config.baseURL || "https://api.etherscan.io/api";
   }
 
+  async getContract(address: string): Promise<ContractData | null> {
+    let url = this.baseURL + '?module=contract&action=getsourcecode&address=' + address;
+    if (this.apiKey) url += "&apikey=" + this.apiKey;
+
+    const r = await fetchJSON(url);
+    if (r.status === "0") {
+        if (r.result === "Contract source code not verified") return {abi: [], name: '', evmVersion: 'Default', compilerVersion: '', runs: 0};
```

Comment:
Should this return `null`?

Also same below for Sourcify.

If not, should we change the return type to not be nullable? Or is there another condition where it may return null?